src/it/scala/org/zalando/spearheads/innkeeper/routes/GetCurrentRoutesSpec.scala

```diff
@@ -0,0 +1,95 @@
+package org.zalando.spearheads.innkeeper.routes
+
+import java.time.LocalDateTime
+
+import akka.http.scaladsl.model.StatusCodes
+import org.scalatest.{BeforeAndAfter, FunSpec, Matchers}
+import org.zalando.spearheads.innkeeper.api.RouteOut
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecTokens.{INVALID_TOKEN, READ_TOKEN, WRITE_STRICT_TOKEN}
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecsHelper._
+import org.zalando.spearheads.innkeeper.routes.RoutesRepoHelper.{deleteRoute, insertRoute, recreateSchema}
+import spray.json.pimpString
+import spray.json.DefaultJsonProtocol._
+import org.zalando.spearheads.innkeeper.api.JsonProtocols._
+/**
+ * @author dpersa
+ */
+class GetCurrentRoutesSpec extends FunSpec with BeforeAndAfter with Matchers {
+
+  val routesRepo = RoutesRepoHelper.routesRepo
+
+  describe("get /current-routes") {
+    describe("success") {
+      val token = READ_TOKEN
+
+      before {
+        recreateSchema
+      }
+
+      it("should return the correct routes") {
+        val createdAt = LocalDateTime.now()
+        val activateAt = createdAt.minusDays(1)
+        insertRoute("R1", createdAt = createdAt, activateAt = activateAt)
+        insertRoute("R2", createdAt = createdAt, activateAt = activateAt)
+
+        val response = getSlashCurrentRoutes(token)
+
+        response.status should be(StatusCodes.OK)
+        val entity = entityString(response)
+        val routes = entity.parseJson.convertTo[Seq[RouteOut]]
+        routes.size should be(2)
+      }
+
+      it("should return the correct routes for a more complex scenario") {
+        val createdAt = LocalDateTime.now()
+        val activateAt = createdAt.minusDays(1)
+
+        insertRoute("R1", createdAt = createdAt, activateAt = activateAt)
+        insertRoute("R2", createdAt = createdAt, activateAt = createdAt.plusMinutes(5))
+        insertRoute("R3", createdAt = createdAt, activateAt = activateAt)
+        insertRoute("R4", createdAt = createdAt, activateAt = activateAt)
+        insertRoute("R1", createdAt = createdAt, activateAt = activateAt)
+        insertRoute("R3", createdAt = createdAt, activateAt = activateAt)
+        deleteRoute(5)
+
+        val response = getSlashCurrentRoutes(token)
+
+        response.status should be(StatusCodes.OK)
+        val entity = entityString(response)
+        val routes = entity.parseJson.convertTo[Seq[RouteOut]]
+        //routes.size should be (3)
+        routes.map(_.id).toSet should be (Set(1, 4, 6))
```

Comment:
is it possible to go assert against the route name here? Like `... should be (Set("R1", "R4", "R3"))`? I think this would make the intention of the test a lot more clear. Right now, I don't really know where 1, 4 and 6 come from. I assume, they rely on the fact, that ids are auto generated and incremental. This is an implementation detail and make the test unnecessarily hard to follow